pkg/workflow_template.go

```diff
@@ -49,6 +55,26 @@ func createWorkflowTemplateVersionDB(runner sq.BaseRunner, workflowTemplateID ui
 	return
 }
 
+// updateWorkflowTemplateVersionDB will update a WorkflowTemplateVersion row in the database.
+func updateWorkflowTemplateVersionDB(runner sq.BaseRunner, wtv *WorkflowTemplateVersion) (err error) {
+	pj, err := json.Marshal(wtv.Parameters)
+	if err != nil {
+		return
+	}
+	_, err = sb.Update("workflow_template_versions").
+		SetMap(sq.Eq{
+			"manifest":   wtv.Manifest,
+			"created_at": wtv.CreatedAt,
+			"is_latest":  wtv.IsLatest,
+			"parameters": pj,
+			"version":    wtv.Version,
```

Comment:
@aleksandrmelnikov 

I don't think we need to update `created_at`, that shouldn't change.
Same for `version`.

Reply:
Agreed.